README.md

```diff
@@ -88,6 +88,7 @@ a more lenient and customizable variant of JSON. Complete the following items in
     `CANVAS` | `API_CLIENT_SECRET` | The client secret for authenticating to the API Directory.
     `CANVAS` | `CANVAS_URL` | The Canvas instance URL to be used as the base URL for API requests that use the `CANVAS TOKEN`.
     `CANVAS` | `CANVAS_TOKEN` | The Canvas token used for authenticating to the API when not using the U-M API Directory.
+    `CANVAS` | `SUPPORTED_LTI_TOOLS` | The tool definitions supported by the CANVAS_LTI job. See the env_blank for a sample of a configuration.
```

Comment:
Not related to your PR, but can you fix the end backtick for the `kaltura_categories_full_name_in` variable below? It's using a single quote at the end now.

Reply:
Yup, fixed!